Bio/PDB/MMCIF2Dict.py

```diff
@@ -84,7 +84,7 @@ def _splitline(self, line):
                     quote_open_char = None
                     in_token = False
                     yield line[start_i:i]
-            elif c == "#" and not quote_open_char:
```

Comment:
The change is well-tested but it'd be nice to have a comment here or even break up the conditional into some descriptive chunks like "at_beginning_of_line".